### PR TITLE
Another design for activities

### PR DESCRIPTION
**Pull request by bhollis**
I worked up an alternate design for Activities that I think is a bit more in line with our other views (like inventory and record books). What do you think?

screen shot 2017-07-09 at 12 16 24 am

### CODE REVIEW

**Comment by bhollis** [comment]
I will note that this one is a bit harder to cram into a portrait phone orientation. The one we have now gets narrow more gracefully:

screen shot 2017-07-09 at 12 23 16 am

**Comment by SunburnedGoose** [comment]
I like it but it gets wispy around the nightfall section.  Lots of blank space around there.

**Comment by bhollis** [comment]
Yeah, I wasn't sure what to do there - one idea was to replace the single dot with an objective style complete/incomplete checkbox, and then grid up the perks. Thoughts?

**Comment by kyleshay** [comment]
Looks good! Could add the collapse sections in to match the other designs also for people who don't care about specific raids or something. Especially if we do add in the rest of the activities

I like the idea of the modifiers looking more like a talent grid, showing details on hover/click, maybe?

**Comment by kyleshay** [comment]
Maybe the modifiers could show in the title bar floated to the right near the word STRIKE as just the icons, then on hover show details? That way each section just has the circles. 

Instead of center aligning the circles maybe try aligning them to the right? I do like this layout as it does align with the inventory page.

**Comment by kyleshay** [comment]
We also lost the category text - for example for *The Nexus* it was in the *Nightfall Strike* category. Not sure if it's really that important though?

[example](https://user-images.githubusercontent.com/424158/27902339-45f5e63a-61ea-11e7-9c94-ff5cc2bf51aa.png)

File: src/scripts/services/dimDefinitions.factory.js [hunk 1]
```diff
@@ -13,16 +13,17 @@ const lazyTables = [
   'VendorCategory',
   'RecordBook',
   'ActivityCategory',
-  'ScriptedSkull'
+  'ScriptedSkull',
```
Comment (delphiactual) [inline src/scripts/services/dimDefinitions.factory.js]: ScriptedSkull can be removed

**Comment by bhollis** [comment]
> Maybe the modifiers could show in the title bar floated to the right near the word STRIKE as just the icons, then on hover show details? That way each section just has the circles.

Might be tough to wrangle for mobile. I can try it in a second pass.

> Instead of center aligning the circles maybe try aligning them to the right? 

I tried that, but it looks kind of weird...

> Maybe we could use something else instead of the circles. I wonder if we could show the rewards based on each circle on click/hover? 

Yeah we can! I'll file an issue.

> We also lost the category text - for example for The Nexus it was in the Nightfall Strike category. Not sure if it's really that important though?

Huh, I don't know where that was coming from. On Beta right now it shows "The Nexus" and "Siva Heroic Strike" which is different from your screenshot. I don't think #1912 would have changed that though...

**Comment by bhollis** [comment]
Right-aligned bubbles:
screen shot 2017-07-10 at 7 03 58 pm

**Comment by bhollis** [comment]
I've added collapsing.

**Comment by delphiactual** [comment]
> We also lost the category text - for example for The Nexus it was in the Nightfall Strike category. Not sure if it's really that important though?

> Huh, I don't know where that was coming from. On Beta right now it shows "The Nexus" and "Siva Heroic Strike" which is different from your screenshot. I don't think #1912 would have changed that though...

This happened during the i18n for activity name/skulls I didn't notice it because the raid names translated correctly, but something is weird with the ActivityTypes for Nightfall and Weekly Heroic in the definitions...

**Comment by bhollis** [comment]
@delphiactual I'm pulling the activity types from the definitions, but they're both coming back as "Strike".

**Comment by delphiactual** [comment]
@bhollis I know inside the raw activity the have "Nightfall Strike" and "Weekly Heroic Strike" but in the definition they do not... I looked through the manifest for about 2 hours last night trying to track them down....

**Comment by kyleshay** [comment]
@delphiactual - one way is we could `activityTypeHash` from the activity and look it up in `DestinyActivityDefinition`, and from there use the `friendlyUrlId` which maps to `friendlyName` in `DestinyActivityModeDefinition`. Unfortunately `DestinyActivityModeDefinition. modeName`is plural though: `Nightfall Strikes` :/ haha. 

Might be better off just having a hardcoded translation for "Nightfall".

**Pull request merged**